pkg/cmd/openshift/openshift.go

```diff
@@ -81,6 +81,8 @@ func CommandFor(basename string) *cobra.Command {
 		cmd = kubernetes.NewCommand(basename, basename, out)
 	case "origin", "atomic-enterprise":
 		cmd = NewCommandOpenShift(basename)
+	case "pod-diagnostics":
```

Comment:
This is in the context of firing up a pod to run diagnostics from within the container. The code has it invoked as "pod-diagnostics".

It would be easy to have instead "openshift ex diagnostics pod" or something like that. The problem would be that it's discoverable by users, but not intended for users to ever use.